frontend/src/app/modify-exercise/modify-exercise.component.ts

```diff
@@ -26,7 +26,8 @@ export class ModifyExerciseComponent {
   difficultyRange = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 
   isValid(): boolean {
-    return this.exercise.name.length > 0;
+    const nameCheck = this.exercise.name.length > 0 && this.exercise.name.length <= 20;
```

Comment:
**suggestion:** for boolean values use a format where the first word is "is", "are", "has" or so on. E.g. `isNameValid` in this case